cmd/bbolt/command_check.go

```diff
@@ -9,16 +9,19 @@ import (
 	"go.etcd.io/bbolt/internal/guts_cli"
 )
 
+var pageId uint64
```

Comment:
Please try not to use global variable. Please refer to other commands on how they manage the parameters.

Reply:
fixed 👍🏽 ptal